core/plugins/org.fusesource.ide.maven/META-INF/MANIFEST.MF

```diff
@@ -21,5 +21,6 @@ Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.m2e.refactoring,
  org.eclipse.m2e.scm,
  org.eclipse.m2e.maven.runtime;bundle-version="1.6.0",
- org.eclipse.m2e.maven.runtime.slf4j.simple;bundle-version="1.6.0"
+ org.eclipse.m2e.maven.runtime.slf4j.simple;bundle-version="1.6.0",
+ org.sonatype.tycho.m2e;bundle-version="0.9.0";visibility:=reexport
```

Comment:
I think that the visibility should not be reexport. If another plugin require sit it needs to include it in required dependency